### PR TITLE
negar/style_guide

### PR DESCRIPTION
**Pull request by negarn**
write styleguide for sass

### CODE REVIEW

File: src/sass/README.md [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+}
+```
+
+
+---
+
+### SVG
+
+<a id="svg-template-versus-sass"></a>
+**[Template versus Sass:](#svg-template-versus-sass)** Add SVGs as components if you need to add classes to modify them in different themes. Otherwise you may import them in Sass.
```
Comment (zakibinary) [inline src/sass/README.md]: Just to add that we can also import the `SVG` directly to `Component` from `src/images` with  something like this `import SomeIcon from 'Images/app_2/header/some_icon.svg';`.

Comment (negarn) [reply src/sass/README.md]: do we need to specify that? isn't that how we import all components anyway?

Comment (zakibinary) [reply src/sass/README.md]: In this use case, we don't need to embed the `SVG` inside a `JSX` file, so we just directly import from `src/images/app_2`. This is useful if we want to have the `SVG` inline but don't need to pass `classes` to it.

Comment (negarn) [reply src/sass/README.md]: I see what you mean. I'll add it.

File: src/sass/README.md [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+```
+
+Keep all common variables in the [constants.scss](https://github.com/binary-com/binary-static/blob/master/src/sass/app_2/_common/base/constants.scss) file.
+
+---
+
+### Units
+
+<a id="units-flexibility"></a>
+**[Flexibility:](#units-flexibility)** If flexibility is needed, for example for font-size, use units such as `rem`, `vh`, `vw`, `fr`, and only use `px` if it's supposed to be a fixed value.
```
Comment (OskarAhl) [inline src/sass/README.md]: I can live without it if no one agrees but I find using scalable rems as a base unit a convenient way of making a responsive layout 😄
```
html {
    font-size: 62.5%; /* 1rem = 10px */
    box-sizing: border-box;
    @include respond(big-desktop) {
        font-size: 75.5%; /* 1rem = 12px */
    }
    @include respond(tab-land) {
        font-size: 50%; /* 1rem = 8px */
    }
    @include respond(tab-port) {
        font-size: 47%; /* 1rem = 7.5px */
    }
    @include respond(phone-xl) {
        font-size: 38%; /* 1rem = 6px */
    }
    @include respond(phone-lg) {
        font-size: 35%; /* 1rem = 5.5px */
    }
}
```
(but can still use the other units for exceptions)

Comment (ashkanx) [reply src/sass/README.md]: @OskarAhl Actually this is what I discussed with Negar and Kelly. my suggestion is we use `rem` and components will be relative to the root. then inside the container, we use `em` which will be relative to its component.

Comment (kellybinary) [reply src/sass/README.md]: thanks @OskarAhl, will add this to my PR :)

**Pull request merged**